int16 operation
uint16 A_x
uint32 x_speed
uint16 A_y
uint32 y_speed
uint16 A_z
uint32 z_speed
